docs/dev/api.md

```diff
@@ -24,45 +24,47 @@ You can check the current accessibility of any Sauce Labs system on the [Sauce L
+The Sauce Labs APIs are organized around REST. Each endpoint is structured as a resource-oriented URL that accepts inline query parameters and form-encoded request bodies, then returns JSON-encoded responses.
 
 Each endpoint is constructed from a `{base-url}` prefix that is based on the data center associated with the Sauce Labs account for which the request is relevant, plus the latests version for the given method. The following table provide the base URLs for each data center.
 
 |Data Center|API Base URL|
-|---|-------|
+|:---|:-------|
 |US|`https://api.us-west-1.saucelabs.com/`|
 |Europe|`https://api.eu-central-1.saucelabs.com/`|
+|APAC Southeast|`https://api.apac-southeast-1.saucelabs.com/`|
+
 
 ### Authentication
 
-The Sauce Labs API uses API keys to authenticate requests. You can view and manage your API key in the Sauce Labs [User Settings](https://app.saucelabs.com/user-settings) page.
+:::tip
+The request examples throughout the API documentation utilize variables in place of Sauce Labs authentication credentials. Consider [setting your credentials as environment variables](/basics/environment-variables/) so that you can simply copy, paste, and run the code examples rather than manually passing your credentials for each call.
+:::
+
+The Sauce Labs API uses API keys to authenticate requests. You can view and manage your API key under your [Sauce Labs User Settings](https://app.saucelabs.com/user-settings).
 
 Authentication to the API is performed via [HTTP Basic Auth](http://en.wikipedia.org/wiki/Basic_access_authentication). Provide your username and API key as the basic auth username and password values, respectively. All requests must be made over HTTPS. Calls made over HTTP or without proper authentication will fail.
 
 You can provide your authentication credentials as inline parameters of your request or using a Basic Header.
 
-```curl "Inline Authenticated Request Example"
+```bash title="Inline Authenticated Request Example"
 curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 --request GET 'https://api.us-west-1.saucelabs.com/team-management/v1/teams'
 ```
 
-```curl "Header Authenticated Request Example"
-curl -L -X GET 'https://api.us-west-1.saucelabs.com/team-management/v1/users/' -H 'Authorization: Basic' -U <SAUCE_USERNAME> -K <SAUCE_ACCESS_KEY>
-```
-
-The request examples throughout the API documentation utilize variables for the authentication credentials. Consider setting your credentials as environment variables using the following commands so you can simply copy the request samples and run them without having to manually pass your credentials for each call.
-
-```jsx "Set Credential Environment Variables"
-$ export SAUCE_USERNAME=<your.user.name>
-$ export SAUCE_ACCESS_KEY=<your access key>
+```bash title="Header Authenticated Request Example"
+curl -L -X GET 'https://api.us-west-1.saucelabs.com/team-management/v1/users/' \
+-H 'Authorization: Basic' \
+-u $SAUCE_USERNAME \
```

Comment:
@sweeneyskirt-sl I wanted to check on the casing for these flags, since normally they're lowercase

Reply:
Hi @kimsauce -- Thanks for the heads up. I will look into it. I feel like references to the credential variables are always all caps, no? See https://docs.saucelabs.com/basics/environment-variables/ for reference.